src/platform/forms/components/review/PreSubmitSection.jsx

```diff
@@ -148,6 +148,11 @@ export function PreSubmitSection(props) {
               >
                 privacy policy
                 <span className="sr-only">opens in a new window</span>
+                <i
+                  className="fas fa-arrow-up-right-from-square"
+                  aria-hidden="true"
+                  role="img"
+                />
```

Comment:
Just FYI that the component library now contain a [va-statement-of-truth web component](https://design.va.gov/storybook/?path=/docs/uswds-va-statement-of-truth--default) that could replace this markup:

https://github.com/department-of-veterans-affairs/vets-website/blob/bc788e4350d2f5f509b85d5b53bb9a3fb1276df7/src/platform/forms/components/review/PreSubmitSection.jsx#L131-L211

Reply:
@tlei123 I saw that you marked this as resolved which is not a big deal but if there's an incompatibility issue or maybe the `va-statement-of-truth` component doesn't work for this use case, do you mind mentioning what that might be? The design system needs that kind of feedback to improve and iterate on the product.